reference/v8.json

```diff
@@ -158,17 +158,17 @@
     "maxzoom": {
       "type": "number",
       "default": 14,
-      "doc": "Maximum zoom to preserve detail at."
+      "doc": "Maximum zoom level at which to create vector tiles (higher means greater detail at high zoom levels but slower performance)."
```

Comment:
Eh, I'm not even sure they are significant enough to be written out. Let's just drop "but slower performance" part here (or everything inside the parentheses) and merge.